examples/bert/yamls/finetuning/glue/mcloud_run.yaml

```diff
@@ -8,7 +8,7 @@
 #
 # When ready, use `mcli run -f yamls/finetuning/glue/mcloud_run.yaml` to launch
 
-run_name: &run_name mosaic-bert-base-uncased-glue-finetuning
+run_name: mosaic-bert-base-uncased-glue-finetuning
```

Comment:
Add cmt that this will populate env var $COMPOSER_RUN_NAME

same for all mcloud yamls

Reply:
Agree.
Ideally we should also mention that $COMPOSER_RUN_NAME will be `run_name` with a unique suffix added to prevent name collision.